packages/holocron/src/holocronModule.jsx

```diff
@@ -172,7 +173,7 @@ export default function holocronModule({
 
         mapModuleStateToProps = ((state) => {
           const moduleState = getModuleState(state);
-          return { moduleState };
+          return { moduleState, ...mapStateToProps(state) };
```

Comment:
Should spread be swapped with existing prop? 
Otherwise may open a door to override `moduleState`, provided by holocron.

Should mapStateToProps follow Redux signature? 
`mapStateToProps(state, ownProps?)` seems like `ownProps` is optional but may be worth to provide it?